app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/navigationui/EmbeddedNavigationActivity.java

```diff
@@ -78,7 +80,8 @@ private void startNavigation(DirectionsRoute directionsRoute) {
         .navigationListener(this)
         .directionsRoute(directionsRoute)
         .shouldSimulateRoute(true)
-        .progressChangeListener(this);
+        .progressChangeListener(this)
```

Comment:
@devotaaabel did you notice `EmbeddedNavigationActivity` being affected by #981?

Reply:
@danesfeder yea I think it was